frontend/src/components/content-types/ContentTypeForm.tsx

```diff
@@ -143,7 +158,11 @@ export const ContentTypeForm: FC<ComponentFormProps<IContentType>> = ({
               startIcon={<AddIcon />}
               variant="contained"
               onClick={() =>
-                append({ label: "", name: "", type: ContentFieldType.TEXT })
+                append({
+                  label: "",
+                  name: "",
+                  type: ContentFieldType.TEXT,
```

Comment:
I suggest we use `generateId()` to generate an uuid for newly appended fields.
```suggestion
                  type: ContentFieldType.TEXT,
                  uuid: generateId(),
```